connectors/sources/jira.py

```diff
@@ -934,6 +949,9 @@ async def _get_issues(self, custom_query=""):
             if self.jira_client.projects == [WILDCARD]
             else projects_query
         )
+
+        self.custom_fields = await self.jira_client.get_jira_fields()
```

Comment:
Can we move this to `get_docs`? `_get_issues` can be called multiple times

Reply:
Yes, that's right! Moved to `get_docs`.